hacking/aws_config/testing_policies/elasticache-policy.json

```diff
@@ -0,0 +1,15 @@
+{# Note that not all EC2 API Actions allow a specific resource #}
```

Comment:
Please don't add a new policy - reuse storage or compute for elasticache (we can only attach 10 policies to a group, so we need to limit the number we have)